File: app/src/main/java/com/blackmirror/dongda/model/request/PhoneLoginRequestBean.java
package com.blackmirror.dongda.model.request;

import com.blackmirror.dongda.model.base.BaseRequestBean;

/**
 * Created by Ruge on 2018-05-04 下午2:21
 */
public class PhoneLoginRequestBean extends BaseRequestBean {
    public String phone_number;
    public String code;
    public String reg_token;
}

File: data/src/main/java/com/blackmirror/dongda/data/model/request/PhoneLoginRequestBean.kt
package com.blackmirror.dongda.data.model.request

/**
 * Created by Ruge on 2018-05-04 下午2:21
 */
class PhoneLoginRequestBean : BaseRequestBean() {
    var phone_number: String? = null
    var code: String? = null
    var reg_token: String? = null
}
